Sources/Core/JavaFileRenderer.swift

```diff
@@ -49,6 +49,56 @@ extension JavaFileRenderer {
         }
     }
 
+    func wrappedClassNameFromSchema(_ param: String, _ schema: Schema) -> String {
+        switch schema {
+        case .array(itemType: .none):
+            return "Object"
+        case let .array(itemType: .some(itemType)):
+            return wrappedClassNameFromSchema(param, itemType)
+        case .set(itemType: .none):
+            return "Object"
+        case let .set(itemType: .some(itemType)):
+            return wrappedClassNameFromSchema(param, itemType)
+        case .map(valueType: .none):
+            return "Object"
+        case let .map(valueType: .some(valueType)):
+            return wrappedClassNameFromSchema(param, valueType)
+        case .string(format: .none),
+             .string(format: .some(.email)),
+             .string(format: .some(.hostname)),
+             .string(format: .some(.ipv4)),
+             .string(format: .some(.ipv6)),
+             .string(format: .some(.uri)):
+            return "String"
+        case .string(format: .some(.dateTime)):
+            return "Date"
+        case .integer:
+            return "Integer"
+        case .float:
+            return "Double"
+        case .boolean:
+            return "Boolean"
+        case let .enumT(enumObj):
+            switch enumObj {
+            case .integer:
+                return "Integer"
```

Comment:
Is this correct since we currently use `@IntDef` / `@StringDef` enums? Doesn't it need the `@enumName` annotation next to the type?

Reply:
Yes, you're right. I was just prototyping at the time. Entire method removed.